lib/fizzy/parser.cpp

```diff
@@ -534,6 +535,37 @@ Module parse(bytes_view input)
         throw parser_error("malformed binary: number of function and code entries must match");
 
     const auto total_func_count = module.imported_function_types.size() + module.funcsec.size();
+    const auto total_global_count =
+        module.imported_globals_mutability.size() + module.globalsec.size();
+
+    // Validate exports.
+    std::unordered_set<std::string_view> export_names;
```

Comment:
Is this much faster than `vector` or what is the reason?

Reply:
With set it is straight-forward to check if element exists.